packages/orbit-components/src/HorizontalScroll/index.tsx

```diff
@@ -76,6 +104,8 @@ const HorizontalScroll = React.forwardRef<HTMLDivElement, Props>(
     const theme = useTheme();
     const scrollEl = scrollWrapperRef.current;
 
+    const TRIGGER_OFFSET = getTriggerOffset(spacing);
```

Comment:
All uppercase constants should be the same value every time, not calculated in runtime